docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -138,40 +138,49 @@ The ability to download recordings for offline viewing will be available in a fu
 
 ### Will Teleport be automatically upgraded with each release?
 
-Teleport Cloud follows the Teleport [Upcoming
-Releases](../../upcoming-releases.mdx) timelines.
+Teleport Team and Teleport Enterprise Cloud are upgraded after each Teleport release
+is delivered. You can see the schedule for Teleport releases on [Teleport Upcoming
+Releases](../../upcoming-releases.mdx).
 
-- Major version upgrades for customer Auth and Proxy Servers occur after the
-  first minor release (for example (=teleport.major_version=).1.0) of Teleport.
+- Major version upgrades for the Teleport Auth Service and Teleport Proxy Service on 
+  Teleport Cloud occur after the first minor release (for example (=teleport.major_version=).1.0) 
+  of Teleport.
   This typically happens within one month of a new major release of Teleport
   becoming available.
 - After the first minor release, subsequent minor and patch upgrades typically
   occur within a week.
 
-<Admonition type="note" title="Minimum agent version">
-    For major version upgrades (e.g. 12.x to 13.x), your Teleport tenant will NOT be automatically upgraded if any
-    Teleport agents connected to the cluster are not compatible (> 1 major version behind).  This is done to ensure
-    backwards compatibility. To get version information on your agents versions
-    see [here](#how-can-i-find-version-information-on-connected-agents).
+For major version upgrades (for example, 12.x to 13.x), your Teleport account is NOT 
```

Comment:
This is correct today, but with automatic upgrades I believe we will no longer hold upgrades due to old agents. Would be good to confirm with @r0mant

Reply:
@r0mant, can you elaborate on what we might need to say here?

Reply:
I can't see this paragraph fully anymore cause I assume it got removed but the current changeset looks good to me. Just left one comment.